src/test/java/core/basesyntax/storage/StrategyAddTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.storage;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.FruitRecordDto;
+import core.basesyntax.servise.Operation;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StrategyAddTest {
```

Comment:
test some incorrect data as well, for example invalid input

Reply:
we came to an agreement